ludwig/features/numerical_feature.py

```diff
@@ -47,12 +47,32 @@ def __init__(self, feature):
 
     preprocessing_defaults = {
         'missing_value_strategy': FILL_WITH_CONST,
-        'fill_value': 0
+        'fill_value': 0,
+        'normalization_strategy': None
     }
 
     @staticmethod
     def get_feature_meta(column, preprocessing_parameters):
-        return {}
+        if preprocessing_parameters['normalization_strategy'] is not None:
+            if preprocessing_parameters['normalization_strategy'] == 'zscore':
+                return {
+                    'mean': column.astype(np.float32).mean(),
+                    'std': column.astype(np.float32).std()
+                }
+            elif preprocessing_parameters['normalization_strategy'] == 'minmax':
```

Comment:
Followed PEP8 style guide everywhere except this line. Couldn't get the character count to below 80. @w4nderlust

Reply:
you can wrap the elif condition in brackets and split the long line (without using '\')

Reply:
actually, even simpler, let's use `normalization` instead of `normalization_strategy` everywhere ;)

Reply:
Changed the `preprocessing_feature` to `normalization` :)